tests/test_chat.py

```diff
@@ -0,0 +1,84 @@
+from unittest.mock import MagicMock
+
+import pytest
+
+from autogpt.agent import Agent
+from autogpt.config.ai_config import AIConfig
+from autogpt.llm.chat import chat_with_ai
+
+
+@pytest.fixture
```

Comment:
should this fixture live in the `conftest.py`? there might be a agent fixture already defined therem, if so lets just use that other fixture